packages/react/src/Bento/Bento.module.css

```diff
@@ -0,0 +1,333 @@
+.Bento {
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  grid-auto-rows: var(--base-size-40);
+  grid-gap: var(--brand-Bento-gap-small);
+  position: relative;
+}
+
+.Bento__Item {
+  background-color: var(--brand-Bento-background);
+  border-radius: var(--brand-Bento-Item-borderRadius-small);
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr;
+  overflow: hidden;
+  border: 1px solid var(--base-color-scale-gray-3);
```

Comment:
Do you know if there's a reason this needs a special color, and not one of the default border colors?

Reply:
This is all still draft so I was just doing mock-ups :) But it will use a default border color and if not I will ask design about it.